README.md

```diff
@@ -1,70 +1,172 @@
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
 
 ### Prerequisites
-HTML & CSS
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
 ### Setup
-git clone <repocitory URL>
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
 ### Install
 
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
 ### Usage
 
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
 ### Run tests
 
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
 ### Deployment
 
+You can deploy this project using:
 
+<!--
+Example:
 
-## Authors
+```sh
 
-👤 **Author1**
+```
+ -->
 
-- GitHub: [@githubhandle](https://github.com/dselasea)
-- Twitter: [@twitterhandle](https://twitter.com/danielsanyidoho)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/danielsanyidoho)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🤝 Contributing
+<!-- AUTHORS -->
 
-Contributions, issues, and feature requests are welcome!
+## 👥 Authors <a name="authors"></a>
 
-Feel free to check the [issues page](../../issues/).
+> Mention all of the collaborators of this project.
 
-## Show your support
+👤 **Author1**
 
-Give a ⭐️ if you like this project!
+- GitHub: [@dselasea](https://github.com/dselasea)
+- Twitter: [@danielsanyidoho](https://twitter.com/danielsanyidoho)
+- LinkedIn: [Daniel Selase Anyidoho](https://www.linkedin.com/in/danielsanyidoho/)
 
-## Acknowledgments
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+<!-- LICENSE -->
 
-## 📝 License
+## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
\ No newline at end of file
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Great work, you are almost done with this project 👏. Currently, when I click the link titled _**MIT**_ in the README.md file, it redirects me to an error page (check the image attached below). 

![image](https://user-images.githubusercontent.com/77106962/211873242-2535b75e-6bfe-47d5-bf91-a81ea22bc299.png)

#### To resolve this issue, kindly add a valid **_LICENSE_**. You can follow the instructions on the right side of the article attached below to set up a valid LICENSE
https://choosealicense.com/licenses/mit/. I would advise that you name the file _**LICENSE**_. 💻 😄